host/ProtocolGateway.Host.Common/Bootstrapper.cs

```diff
@@ -45,7 +46,7 @@ public class Bootstrapper
         readonly IotHubClientSettings iotHubClientSettings;
         readonly IMessageAddressConverter topicNameConverter;
 
-        public Bootstrapper(ISettingsProvider settingsProvider, ISessionStatePersistenceProvider sessionStateManager, IQos2StatePersistenceProvider qos2StateProvider)
+        public Bootstrapper(ISettingsProvider settingsProvider, ISessionStatePersistenceProvider sessionStateManager, IQos2StatePersistenceProvider qos2StateProvider, List<string> inboundTemplates = null, List<string> outboundTemplates = null)
```

Comment:
> = null [](start = 196, length = 7)

code style: pls add a separate overload without inboundTemplates and outboundTemplates parameters instead of using default values here.